### DISCUSSION TITLE
how to integrate fortigate 7.0.12 with wazuh

### DISCUSSION THREAD

**Discussion by AjDenning** [wazuh/wazuh#20273] [Q&A]
Hey everyone just reaching out as i have found things for fortios 6.0 but needing to find out what all or how to get our Fortigate firewalls that are all on 7.0.12 or 7.0.13 to push logs to Wazuh. Would the rulesets and config for ossec be the same as 6.0.

**Comment by erikhsnr** [comment]
You could try to integrate with the Wazuh Connector for FortiSOAR Playbooks. 

https://docs.fortinet.com/document/fortisoar/1.0.0/wazuh-siem/347/wazuh-siem-v1-0-0
